### PR TITLE
[Docs] enhance: use flex layout for landing page cards

### LINKED ISSUE #76: [Docs] Landing Page: Use all flex layout, not grid layout
#### Current State
The homepage (aka the landing page) of this doc site has a mixture of two different types of container layouts.

#### Desired State

1. Ideally, only flex layout is used.
2. Rearrange the layout slightly (see mockup).

Screenshot 2023-11-17 at 1 47 20 PM 1

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by SamIsTheFBI**
**Notes for Reviewers**

This PR fixes #76. Flex layout is used and all instances of grid are removed.

|image|image|
|--|--|
|<div align="center"> **Before** </div>|<div align="center"> **After** </div>|

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by iArchitSharma** [comment]
@SamIsTheFBI Thanks that looks good, can also fix th eimage like we have in figma :
image

File: content/en/_index.md [hunk 1]
```diff
@@ -39,55 +39,62 @@ onmouseover="changeImage('meshery', 'images/logos/meshery-light.svg')" onmouseou
...
+  <div class="landing-card">
     <div class= "playground-section playground-logo">
       <span class="product-head">Meshery playground</span>
       <p>Use Meshery Playground to explore a new way of DevOps - visual and collaborative configuration management for your infrastructure.</p>
     </div>
   </div>
 </a></div>
 
-<div class="box">
+<div class="box2">
```
Comment (leecalcote) [inline content/en/_index.md:56]: @SamIsTheFBI thanks! 

Employ your imagination here for a better variable name.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by SamIsTheFBI** [comment]
Hi @iArchitSharma could you tell me where I can get this figma file for reference?

**Comment by iArchitSharma** [comment]
@SamIsTheFBI   https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?type=design&node-id=21084%3A6224&mode=design&t=fpzsSKW93n2kr9j0-1

**Comment by SamIsTheFBI** [comment]
@iArchitSharma I have made some changes to make the forum card image look a lot like the one in Figma. Please check it.

Also, @leecalcote I changed some classnames. Please tell me if they're desirable.

**Comment by saurabh100ni** [comment]
Hi @SamIsTheFBI ,
Thanks for your PR! I encourage you to share this in the Website Meeting tomorrow at 6:30 PM IST. Share your screen, and walk us through it. It's cool to take and share feedback. Check the [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit) here to include this PR.

You're welcome at the meeting

Check to [community calendar](https://layer5.io/community/calendar) for more details of meetings at Layers.

File: assets/scss/_landing_project.scss [hunk 2]
```diff
@@ -137,22 +123,65 @@
...
+  flex-direction: column;
+  justify-content: space-between;
+
+  padding-right: 0;
+  padding-bottom: 0;
+  max-height: 620px;
+  overflow: hidden;
+}
+
+.hidden {
```
Comment (leecalcote) [inline assets/scss/_landing_project.scss:160]: @SamIsTheFBI is this used anywhere?

**Comment by leecalcote** [comment]
@iArchitSharma review this carefully. Make any changes necessary and merge.

File: assets/scss/_landing_project.scss (hunk 2 above)
Comment (SamIsTheFBI) [reply assets/scss/_landing_project.scss]: oh right. I was using the `hidden` class as a console.log thing. I'd hide elements and try to figure out what's happening in the code.

This commit [0a93c93](https://github.com/layer5io/docs/pull/156/commits/0a93c933290ce073ca052563f327122759471c4a) removes it and any other comments.

**Review by iArchitSharma — APPROVED** [review]
LGTM thanks @SamIsTheFBI

**Comment by SamIsTheFBI** [comment]
Okay @iArchitSharma  but how should I review it? Like, what am I supposed to do? I don't know what's the process in Github 😅

**Pull request merged**